src/apis/index.js

```diff
@@ -113,11 +115,11 @@ export async function getContributors() {
   try {
     let res = await Promise.all(
       repos.map(repo =>
-        ajax.get(`https://api.github.com/repos/fossasia/${repo}/contributors`),
+        axios.get(`https://api.github.com/repos/fossasia/${repo}/contributors`),
```

Comment:
try using `ajax` instead of `axios`. As `ajax` function is based on `axios` with some modified headers, it is not the standard ajax that is directly imported form some library

Reply:
In `ajax` file, it is adding the access token to the url for get method, due to which url changes for the api of contributors and returns an empty array.
Could you please elaborate, what's the problem in using axios here?

Reply:
try using 
```
repos.map(repo => {
  const settings={ isTokenRequired: false };
  return axios.get(url,null,settings)
})
```

you can see more settings at 

https://github.com/fossasia/susi.ai/blob/5c698dba1258d6eb17296cfb3f39451ef9783033/src/helpers/ajax.js#L13

Reply:
We can do one more thing, we can add if condition for this url so that token would not be add in this url.

Reply:
Which method should I prefer?